whois-update/src/main/java/net/ripe/db/whois/update/handler/SingleUpdateHandler.java

```diff
@@ -102,7 +102,7 @@ public void handle(final Origin origin, final Keyword keyword, final Update upda
         }
 
         if (action == Action.DELETE && originalObject == null) {
-            updateContext.addMessage(update, UpdateMessages.objectNotFound(updatedObject.getFormattedKey()));
+            updateContext.addMessage(update, UpdateMessages.objectNotFound(update.getSubmittedObject().getFormattedKey()));
```

Comment:
So the error message will contain the key as the user submitted it?

Reply:
Yes, like it was before

Reply:
Great, thanks